### ISSUE TITLE
Filter on Results table is blank

### ISSUE THREAD

**Issue by guyfromwy** [dbeaver/dbeaver#6877]
I was unable to find a way to re-open a bug that has not been resolved but has been closed and just commenting on it does not seem to mark it for attention. Sorry for the duplicate bug report, so this is the same as:
DBeaver 6.1.5: Filters in results Grids are not available #6612

This remains an issue that I am still seeing in 6.2.1. 

Description:
For each column in the Results View table, I am unable to use the Filter to see what is listed in the column. 

![image](https://user-images.githubusercontent.com/26490963/65893705-a622a080-e365-11e9-9f6a-15079190c54d.png)
![image](https://user-images.githubusercontent.com/26490963/65893720-ade24500-e365-11e9-8c14-dba75408e66d.png)

**Comment by uslss** [comment]
Hi @guyfromwy ,
unfortunately I cannot reproduce the issue on 6.2.1.
Could you add some info like db type, permissions?
Probably there is something  appears in log (https://github.com/dbeaver/dbeaver/wiki/Log-files)?

**Comment by guyfromwy** [comment]
This is an Oracle database that I have query-only access.
I'm glad you brought that up. I quickly tested a couple other scenarios. When I query a .csv file, I do get the filter list. I also get a filter list when I query a slqlite database. Interesting.

I don't see anything in the logs that points to a potential problem, though. Of the three log files (.log, dbeaver_sql_20190924.log, and dbeaver-debug.log), the only one that changed when I ran a simple select * was dbeaver_sql_20190924.log. This is what showed up:


!ENTRY org.jkiss.dbeaver.model 1 920 2019-10-02 10:55:55.607
!MESSAGE SELECT * FROM rtvccoe
!SUBENTRY 1 org.jkiss.dbeaver.model 1 0 2019-10-02 10:55:55.610
!MESSAGE SUCCESS [-1]

The other files did not get updated when I ran that query. I'm not saying there isn't something further up in one of the log files, like at initialization, but I'm not seeing it if there is. Admittedly, I only gave the logs a cursory glance to try to find it and I honestly don't know specifically what I'm looking to find.

Let me know what else I can provide.
Thanks!
